docs/simulating_with_the_mvs.rst

```diff
@@ -72,10 +72,17 @@ this can really only be recommended for advanced users.
 
 There can only be a single Json file in your input folder.
 As some parameters in the Json file link to a time series provided as a CSV,
-the folder "time_series" should be present in your input folder
-and provide all necessary input data that is an timeseries format.
-This can include for example PV generation time series demand time series.
+the folder "time_series" should be present in your input folder, as clarified in the next section.
+
 
+Time series: time_series folder
```

Comment:
@smartie2076  could you please, during your review, check this section, as I might not be up-to-date. thank you

Reply:
Yeah, this part is interesting. Currently, we actually devided for the EPA that the timeseries should already be included in the json file. This is currently not the case. Still, maybe you can move this section below the section that describes the csv inputs?

Reply:
Okay, so I also deleted the mentioning of json files in this section.